jujugui/static/gui/src/app/assets/javascripts/zip.js

```diff
@@ -26,783 +26,780 @@
  EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-YUI.add('zip', function(Y) {
```

Comment:
Can this file be removed entirely and require it from the npm installed version?

Reply:
Yep, I'll do it as a followup. Tracked in #3245.